hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSenderTest.java

```diff
@@ -332,10 +338,34 @@ public void shouldDeliverToNewSenderAfterModifiedTimeout() {
 
         // when
         sender.updateSubscription(subscriptionWithModfiedTimeout);
-        sender.sendMessage(message);
+        sender.sendAsync(message);
 
         // then
-        verify(otherMessageSender).send(message);
+        verify(otherMessageSender, timeout(1000)).send(message);
+    }
+
+    @Test
+    public void shouldDelaySendingMessageForOneSecond() {
+        // given
+        Subscription subscription = subscriptionBuilderWithTestValues()
+                .withSubscriptionPolicy(subscriptionPolicy().applyDefaults()
+                        .withSendingDelay(500)
```

Comment:
Tests says `shouldDelaySendingMessageForOneSecond` but here we've `500`. Am I missing something?

Reply:
Good catch. Started from one second, changed it later to speed up test. It is fine, but forgot to change method name.